src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -3132,7 +3138,7 @@ inline void rpp_store8_f32pln1_to_i8pln1_avx(Rpp8s* dstPtr, __m256 &p)
 {
     __m256i px1 = _mm256_permute4x64_epi64(_mm256_packus_epi32(_mm256_cvtps_epi32(p), avx_px0), _MM_SHUFFLE(3,1,2,0));
     px1 = _mm256_sub_epi8(_mm256_packus_epi16(px1, avx_px0), avx_pxConvertI8);  /* Pack and add I8 conversion param */
-    _mm256_storeu_si256((__m256i *)(dstPtr), px1);                              /* store the 4 pixels in dst */
+    rpp_storeu_si64((__m128i *)(dstPtr), _mm256_castsi256_si128(px1));          /* store the 4 pixels in dst */
```

Comment:
we are storing 24 values in output in this function before
but now we are storing only 64. Is this correct?

Reply:
Only 1st quarter stores valid values, on storing register to size 256 it lead to output corruptions in remap kernel.